Makefile

```diff
@@ -35,3 +35,22 @@ dependencies:
 
 test:
 	make -C git-webhook test
+
+
+MANIFESTS_DIR ?= deploy/manifests
+CRDS_FILE ?= $(MANIFESTS_DIR)/00-crds.yaml
+
+CERT_MANAGER_VERSION ?= v0.11.0
```

Comment:
Can we detect versions from helm lock-file so that we keep the source of truth in a single place? We should also add a verification step on CI.